modules/rules_java/metadata.json

```diff
@@ -1,14 +1,15 @@
 {
     "homepage": "https://github.com/bazelbuild/rules_java",
     "maintainers": [],
-    "repository": [
```

Comment:
This is accidentally removed, can you please add the "repository" field in the template as well?

Reply:
@meteorcloudy I don't think it's removed, it's added back in line 12.

Reply:
Ah, you're right, thanks!